### DISCUSSION TITLE
The volume bar does not scale and becomes too large.

### DISCUSSION THREAD

**Discussion by jskorlol** [tradingview/lightweight-charts#872] [Offtopic]
If I create a volume bar, it goes like this.

Sometimes this happens , how do I fix it ?

<img width="649" alt="111" src="https://user-images.githubusercontent.com/12728472/137374217-f408f6e2-225a-46c2-acfe-659378ac3a05.png">

**Comment by timocov** [comment]
Could you provide a repro of the issue please?
